### PR TITLE
Create README.md and git action

### LINKED ISSUE #1: Create README.md

### LINKED ISSUE #11: Make Git Action

### PR DESCRIPTION
**Pull request by seohy02**
I created README.md and git action

About git action, I added 'build test' using `./gradlew build` on linux. It is called when pull request is created. Because we have only master branch, I didn't add specific branch.
When someone create a pull request, build test starts.
image

The details are as follows.
image
image
If successful, it is as follows.
image

Review please!
Thanks !!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+1. Select a business card template that you want.
+
+<img src = "https://user-images.githubusercontent.com/68395698/130160475-4242d19d-e795-454e-a782-242ef203ed60.gif" width =30% />
+
+2. Enter your information on your business card and press the plane button. Then the speaker turns on and the listener turns off.
+
+<img src = "https://user-images.githubusercontent.com/68395698/130109455-c5de67ed-545b-460c-ae74-9d4cbb307184.png" width =30% /> <img src = "https://user-images.githubusercontent.com/68395698/130109811-d8a684a5-b967-45cc-944f-64d2121840f0.png" width =30% /> <img src = "https://user-images.githubusercontent.com/68395698/130109759-e812a94b-3d68-4c27-8328-99e62f35280c.png" width =30% />
+
+3. The listener always works when you turn on the app. Press the lower right button and check business cards sent.  That's it!
```
Comment (Jsueeee) [inline README.md]: as far as I know, listeners don't always work.
listener works when SPEAK OFF & LISTEN ON.
@psh2849 ! right???
Let me know if I'm wrong!

Comment (psh2849) [reply README.md]: > as far as I know, listeners don't always work.
> listener works when SPEAK OFF & LISTEN ON.
> @psh2849 ! right???
> Let me know if I'm wrong!

When the app is run, the permission is checked.
And if the permission is allowed, the 'playListen' function is called.
So, It is correct that when you run the app, it automatically listens. 

Like this:
image

Comment (seohy02) [reply README.md]: That's right!  In 2, I wrote that "Then the speaker turns on and the listener turns off." I wrote this sentence "The listener always works when you turn on the app" to mean that there is no specific speaker turn off or turn on button. 
But I think this sentence was ambiguous!! I'll fix it !

Comment (seohy02) [reply README.md]: How about "The listener works except when the speaker turns on." ???

Comment (Jsueeee) [reply README.md]: image

Isn't it off here???

Comment (Jsueeee) [reply README.md]: @seoh02h That's Perfect!

Comment (psh2849) [reply README.md]: If the speaker and listener are turned on at the same time,
it listens to your own device.
So, When you run the app, turn on the listener, and turn off the speaker.
After you make a business card and clicked airplane picture, turn off the listener and turn on the speaker.  

As a result, the listener is not always turn on. but is temporarily turned on when the app is launched.

Comment (Jsueeee) [reply README.md]: @psh2849 Thanks for the detailed answer! 😄

Comment (seoh02h) [reply README.md:26]: It was hard for me to receive fully correct information on my phone,,,,, So I used cardboard saved screenshot of @psh2849 !! Special Thanks!!

**Comment by seohy02** [comment]
It was hard for me to receive fully correct information of business card,,,,, So I used cardboard saved screenshot of @psh2849 !! Special Thanks!!

**Review by sunnnl — COMMENTED** [review]
I have a question.
What is the base source of this PR ?
Because I made directory '.github' too.

**Comment by seohy02** [comment]
> I have a question.
> What is the base source of this PR ?
> Because I made directory '.github' too.

The base source is "euphony-io/here-is-my-card" !! I was worried about that too..
After your PR merged, I think I might have to do a rebase if necessary.

**Pull request merged**